src/schema/v2/__tests__/searchCriteriaLabel.test.ts

```diff
@@ -0,0 +1,375 @@
+      {
+        name: "Rarity",
+        value: "Unique",
+        field: "attributionClass",
+      },
+      {
+        name: "Rarity",
+        value: "Limited edition",
+        field: "attributionClass",
+      },
+      {
+        name: "Rarity",
+        value: "Open edition",
+        field: "attributionClass",
+      },
+      {
+        name: "Rarity",
+        value: "Unknown edition",
+        field: "attributionClass",
+      },
+    ])
+  })
+
+  it("formats medium criteria", async () => {
+    const parent = {
+      additionalGeneIDs: [
+        "painting",
+        "photography",
+        "sculpture",
+        "prints",
+        "work-on-paper",
+        "nft",
+        "design",
+        "drawing",
+        "installation",
+        "film-slash-video",
+        "jewelry",
+        "performance-art",
+        "reproduction",
+        "ephemera-or-merchandise",
+      ],
+    }
+
+    const labels = await resolveSearchCriteriaLabels(parent, _, _, _)
+
+    expect(labels).toIncludeAllMembers([
+      {
+        name: "Medium",
+        value: "Painting",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Photography",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Sculpture",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Prints",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Work on Paper",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "NFT",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Design",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Drawing",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Installation",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Film/Video",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Jewelry",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Performance Art",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Reproduction",
+        field: "additionalGeneIDs",
+      },
+      {
+        name: "Medium",
+        value: "Ephemera or Merchandise",
+        field: "additionalGeneIDs",
+      },
+    ])
+  })
+
+  it("formats price criteria", async () => {
+    const parent = {
+      priceRange: "42-420",
+    }
+
+    const labels = await resolveSearchCriteriaLabels(parent, _, _, _)
+
+    expect(labels).toIncludeAllMembers([
+      {
+        name: "Price",
+        value: expect.anything(), // TODO: fix this placeholder formatting
+        field: "priceRange",
+      },
+    ])
+  })
```

Comment:
In the interest of momentum I'm wiring up the basics here, but deferring the persnickety formatting for size and price to future PRs, since they likely involve some user and/or locale wrangling. (We already have some subtasks [here](https://artsyproduct.atlassian.net/browse/FX-3216) for that, too)